org.uqbar.project.wollok.ui.launch/src/org/uqbar/project/wollok/ui/tests/model/WollokTestResults.xtend

```diff
@@ -52,11 +57,11 @@ class WollokTestResults extends Observable implements WollokRemoteUITestNotifier
 		container.processingManyFiles = processingManyFiles
 		container.mainResource = URI.createURI(containerResource)
 		container.defineTests(newArrayList(tests.map[new WollokTestResult(it)]), this.shouldShowOnlyFailuresAndErrors)
-		this.superContainer.processingManyFiles = processingManyFiles
-		this.superContainer.mainResource = URI.createURI(containerResource)
-		this.superContainer.add(container)
 		
-		this.globalContainer.add(this.superContainer)
+		this.fileContainer.processingManyFiles = processingManyFiles
```

Comment:
¿Es necesario el `this`?, en este proyecto solemos no usarlo cuando no hace falta.